watchtower/lookout/lookout.go

```diff
@@ -118,9 +165,18 @@ func (l *Lookout) watchBlocks(epochs *chainntnfs.BlockEpochEvent) {
 			log.Debugf("Fetching block for (height=%d, hash=%s)",
 				epoch.Height, epoch.Hash)
 
-			// Fetch the full block from the backend corresponding
-			// to the newly arriving epoch.
-			block, err := l.cfg.BlockFetcher.GetBlock(epoch.Hash)
+			// Fetch the full block corresponding to the newly
+			// arriving epoch from the backend.
+			// This is done using the fetchBlockWithRetries
+			// function, which attempts to fetch the block
+			// multiple times in case of temporary failures.
+			// This operation is crucial for keeping the local
+			// representation of the blockchain up-to-date with
+			// the newly arriving epoch, ensuring the system remains
+			// in sync with the backend.
```

Comment:
i think the name of the function is self explanatory. So can leave this comment out or shorten it quite a bit I think